samples/contrib/versioned-pipeline-ci-samples/mnist-ci-sample/create_pipeline_version_and_run.py

```diff
@@ -0,0 +1,52 @@
+import kfp
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--bucket_name', help='Required. gs bucket to store tensorboard', type=str)
+parser.add_argument('--commit_sha', help='Required. commit sha to ', type=str)
+parser.add_argument('--version_name', help='Required. Name of the new version. Must be unique.', type=str)
+parser.add_argument('--package_url', help='Required. pipeline package url', type=str)
+parser.add_argument('--pipeline_id', help = 'Required. pipeline id',type=str)
+parser.add_argument('--gcr_address', help='Required. Your cloud registry path. For example, gcr.io/my-project', type=str)
+parser.add_argument('--host', help='Host address of kfp.Client. Will be get from cluster automatically, type=str, default='')
```

Comment:
Please forgive comments from passerby, I noticed this arg is missing a closing `'`.

Reply:
Thanks for reply! Will modify it in new pull request.